dev_requirements.txt

```diff
@@ -1,7 +1,8 @@
 -r requirements.txt
 
 requests_mock
```

Comment:
It's been a while since I touched this repository, looks like I wasn't into pinning dependencies at that time. I'll come in here and clean this up in a separate issue 👀